addons/website/static/src/js/content/snippets.animation.js

```diff
@@ -858,6 +858,25 @@ registry.socialShare = Animation.extend({
     },
 });
 
+registry.facebookPage = Animation.extend({
+    selector: '.o_facebook_page',
+
+    /**
+     * @override
+     */
+    start: function () {
+        var params = _.pick(this.$el.data(), 'href', 'adapt_container_width', 'height', 'width', 'tabs', 'small_header', 'hide_cover', 'show_facepile');
+        if (params && params.href) {
+            var actualWidth = this.$el.width();
+            params.width = actualWidth > 500 ? 500 : actualWidth < 180 ? 180 : actualWidth;
```

Comment:
This was from specs ? Any reason we are not always using the `actualWidth` ? And can't Facebook iframe adapt automatically without any code ? (Just asking, I am not supposed to take functional decisions).

Reply:
Hello @qsm-odoo 
Actually facebook provides two way to add fb page plugin

1. Add iframe
2. Add facebook javascript in assets (which internally create iframe for you)

In 2nd option we have parameter adapt_container_width which will resize iframe based in parent container (max 500px)
Because we choose first option we need to handle that case manually to avoid responsive issues